src/common/snippets/include/snippets/lowered/expression_factory.hpp

```diff
@@ -24,6 +24,10 @@ class LinearIR::ExpressionFactory {
             return create(loop_begin, params...);
         } else if (const auto loop_end = ov::as_type_ptr<op::LoopEnd>(n)) {
             return create(loop_end, params...);
+        } else if (const auto perf_counter = ov::as_type_ptr<op::PerfCountBeginBase>(n)) {
+            return create(perf_counter, params...);
+        } else if (const auto perf_counter = ov::as_type_ptr<op::PerfCountEndBase>(n)) {
+            return create(perf_counter, params...);
```

Comment:
custom creation functions and its' call should be under SNIPPETS_DEBUG_CAPS

Reply:
Fair enough, fixed